webapp/tests/forms/steps/lotse/test_steuerminderungen.py

```diff
@@ -16,6 +23,184 @@ def step_gem_haushalt():
     return step
 
 
+# PRECONDITIONS
+class TestShowVorsorgePrecondition:
+    def test_if_vorsorge_not_set_then_raise_validation_error(self):
+        data = {}
+        with pytest.raises(ValidationError):
+            ShowVorsorgePrecondition.parse_obj(data)
+
+    def test_if_vorsorge_set_false_then_raise_validation_error(self):
+        data = {'stmind_select_vorsorge': False}
+        with pytest.raises(ValidationError):
+            ShowVorsorgePrecondition.parse_obj(data)
+
+    def test_if_vorsorge_set_true_then_do_not_raise_validation_error(self):
+        data = {'stmind_select_vorsorge': True}
+        try:
+            ShowVorsorgePrecondition.parse_obj(data)
+        except ValidationError:
+            pytest.fail("Should not raise a validation error")
+    
+    def test_step_to_redirect_to_is_set_correctly(self):
+        assert ShowVorsorgePrecondition._step_to_redirect_to == StepSelectStmind.name
```

Comment:
Personally, I find this kind of test has almost negative value. It's testing data, not behaviour by just checking that the value on the class is set to what it is set to! How can it possibly _fail_? The value can be _wrong_, but then it's very likely wrong in the test, too, and if you need to change it, you need to change it in two places, with no benefit.

Reply:
I definitely see you and I also thought about whether it would be a good idea to include them. It just felt weird to me to only test that on the functional level. But I can definitely just delete that.